fixtures/fixtures.js

```diff
@@ -0,0 +1,43 @@
+export const regionsFixture = [
```

Comment:
```suggestion
export const regions = [
```

변수명에 굳이 fixture라고 하실 필요 없어요. 사용하는 곳에서 이후에 fixture가 아닌 데이터를 사용하게 된다면

모두 변경점이 됩니다.

Reply:
- 이 경우 다른 곳에서 변수이름이 겹치는데가 많습니다ㅠㅠ
- 리덕스 스토어에도 위와 같은 이름으로 선언되어있어서 `regions` 와 같이 적용하면 기존 스토어를 수정해줘야 할거 같습니다.
그렇기 때문에 다른 네이밍으로 짓는 것이 필요할 거 같은데 restaurantRegions? regionsMook? regionsData? 어떤식으로 짓는 것이 좋을까요??
- 고민이 됩니다..🙏